### PR TITLE
added provider sndcmp #184

### PR DESCRIPTION
**Pull request by Bibiko**

### CODE REVIEW

**Comment by xrotwang** [comment]
For testing, you'd need to add a test dataset here https://github.com/lexibank/pylexibank/tree/master/tests/repos/datasets and then test the workflow commands in https://github.com/lexibank/pylexibank/blob/master/tests/test_commands.py

File: src/pylexibank/providers/sndcmp.py [hunk 1]
```diff
@@ -0,0 +1,359 @@
+import pathlib
+import zipfile
+import attr
+import json
+import re
+
+from collections import OrderedDict
```
Comment (xrotwang) [inline src/pylexibank/providers/sndcmp.py]: This is also a stdlib import, thus should be in the first block of imports. We also try to always import the base module/package from stdlib, i.e.
```python
import collections
```

File: src/pylexibank/providers/sndcmp.py [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+        replacements=[],
+        separators='',
+        missing_data=('..', '--', '**', '-..', '...'),
+        strip_inside_brackets=False,
+        normalize_unicode='NFC',
+    )
+
+    def __init__(self):
+        self.study_name = self.study_name
+        self.second_gloss_lang = self.second_gloss_lang
```
Comment (xrotwang) [inline src/pylexibank/providers/sndcmp.py]: The above two lines don't do anything, so should be removed.

**Comment by xrotwang** [comment]
Tests are failing on py3.5 - did you run `tox -r` locally?

**Comment by Bibiko** [comment]
I ran `pytest` against py3.7

**Pull request merged**